.github/workflows/publish-to-pypi.yml

```diff
@@ -3,7 +3,7 @@ name: publish 🐍 to PyPI
 on:
   push:
     tags:
-      - "v*"
+      - "[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
isn't the initial v missing here?

Reply:
No. That's the point of this change. The tags will NOT contain the "v" prefix anymore because - if I got [this](https://docs.readthedocs.io/en/stable/versions.html#how-we-envision-versions-working) right - this is what rtd requires to start creating versioned docs.